dts/bindings/gpio/zephyr,gpio-keys.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2022 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+description: Zephyr GPIO KEYS parent node
+
+compatible: "zephyr,gpio-keys"
+
+include: base.yaml
+
+properties:
+    gpios:
+      type: phandle-array
+      required: true
+    label:
```

Comment:
I would sort of expect an integer key code or enumeration here, but it doesn't exist. Is this more of just a "debounce gpio" driver than a "gpio keys" driver?

Reply:
I'd say more of a "debounce gpio" driver.  The linux implementation uses a integer key code as I referenced as an alternative in https://github.com/zephyrproject-rtos/zephyr/issues/52344.  I opted to omit this in favor of using a chosen node to reduce memory usage.

Reply:
See comment above - my one concern is that by using a DT chosen node, we might be implicitly limiting the number of devices capable of reporting a specific input event.